admin_guide/pruning_resources.adoc

```diff
@@ -283,9 +306,39 @@ $ oadm prune images --keep-tag-revisions=3 --keep-younger-than=60m --confirm
 $ oadm prune images --prune-over-size-limit --confirm
 ----
 
+.Using secure or insecure connection
```

Comment:
Perhaps we can use [discrete headings ](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#discrete-headings)here instead?